crates/wasm-mutate/src/lib.rs

```diff
@@ -165,7 +215,8 @@ impl WasmMutate {
         while !mutators.is_empty() {
             let i = rng.gen_range(0, mutators.len());
             let mutator = mutators.swap_remove(i);
-            if let Ok(module) = mutator.mutate(self, &mut rng, &info) {
+            let mut resources = Resources::new(self.fuel);
+            if let Ok(module) = mutator.mutate(self, &mut rng, &info, &mut resources) {
```

Comment:
Rather than adding another parameter to the `mutate` trait method and defining a new `Resource` type, lets just use the currently-unused `WasmMutate::consume_fuel` method.